libs/analytics/WordPressAnalytics/build.gradle

```diff
@@ -20,8 +20,8 @@ android {
     compileSdkVersion 29
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 29
+        minSdkVersion rootProject.minSdkVersion
+        targetSdkVersion rootProject.targetSdkVersion
```

Comment:
Nice to be reading these this way:

- Always in sync across components of the project
- Should not need to change this line anymore